src/main/jbake/content/admin/index.adoc

```diff
@@ -1,9 +0,0 @@
-= TomEE Administration
```

Comment:
Looking at most server sites either 'Docs' or 'Documentation' is consistent with what people expect